Kernel/Net/NetworkTask.cpp

```diff
@@ -117,10 +120,10 @@ void NetworkTask_main(void*)
             handle_arp(eth, packet_size);
             break;
         case EtherType::IPv4:
-            handle_ipv4(eth, packet_size, packet_timestamp);
+            handle_ipv4(eth, packet_size, meta.packet_timestamp);
             break;
         case EtherType::IPv6:
-            // ignore
+            handle_ipv6(eth, packet_size, meta.packet_timestamp, meta.adapter);
```

Comment:
`meta` does not exist at this point in time, you may want to move the refactor commit up in the history